Gopkg.toml

```diff
@@ -107,3 +107,12 @@ required = [
 [[constraint]]
   name = "github.com/pkg/sftp"
   version = "1.10.0"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/metal3-io/cluster-api-provider-baremetal"
+  source = "github.com/openshift/cluster-api-provider-baremetal/pkg/apis"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/metal3-io/baremetal-operator"
```

Comment:
both of these need to be the `github.com/openshift/` versions

Reply:
Should we do this on the fork, or only on my openshift/installer branch?

Reply:
the fork, not just your branch

Reply:
For CAPB, it is pulling from openshift, just name is metal3-io, is that right?

```
[[constraint]]
  branch = "master"
  name = "github.com/metal3-io/cluster-api-provider-baremetal"
  source = "github.com/openshift/cluster-api-provider-baremetal/pkg/apis"
```

It's how others are done:

```
[[constraint]]
  branch = "master"
  name = "sigs.k8s.io/cluster-api-provider-azure"
  source = "https://github.com/openshift/cluster-api-provider-azure.git"
```

Reply:
yes you're right, so it's just baremetal-operator using the wrong source

Reply:
https://github.com/openshift-metal3/kni-installer/pull/114/files